src/applications/accredited-representative-portal/routes.jsx

```diff
@@ -22,10 +25,14 @@ const router = createBrowserRouter(
             {
               path: 'poa-requests',
               element: <POARequestsPage />,
+              loader: poaRequestsLoader,
+              errorElement: <ErrorMessage />,
             },
             {
               path: 'poa-requests/:id',
               element: <POARequestDetailsPage />,
+              loader: poaRequestLoader,
+              errorElement: <ErrorMessage />,
```

Comment:
In the future we can have an error boundary hierarchy, where a generic error message like this is at the top. `POARequestDetailsPage` can handle `404` and rethrow others if the framework works that way (I think it does).

Reply:
yeah this is driven by which errors we let hot routes in the stack so down the road we can control use case vs global by what we handle vs what we throw